src/verter/store/postgres.clj

```diff
@@ -45,8 +43,7 @@
 (defn- record-facts [{:keys [ds schema]}
                      facts]
   (let [sql (make-insert-facts-batch-query schema facts)]
-    (jdbc/execute! ds sql
-                   {:return-keys true :builder-fn jdbcr/as-unqualified-lower-maps})))
```

Comment:
Why remove these options?

Reply:
Those options return the inserted rows.  However those values are not passed on anywhere so they are just extra execution time.

Reply:
![image](https://user-images.githubusercontent.com/33734037/225989873-55ed0fdb-6f65-44c0-9e1e-3a33a4640b3d.png)